### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by Feliix42** [deps-rs/deps.rs#40]
Hey there!

First of all I want to thank you for the enormous work you did for the community with this project! I love deps.rs and am using it quite often as a dashboard to visualize the dependency states of my or other projects.

But I came to notice that this project seems to be somewhat unmaintained right now - which is totally fine, everybody shifts their attention at one point or another to a new project or is unable to find the time to take care of the maintenance! But it would be a bummer to leave this out in the wild to rot. Would you consider opening the project up to the community or a volunteer - should one be found - to take this over?

I'm not saying that this specific person should be me (I'm not sure if I have the time required at hand) but it would be cool if the project would live on.

**Comment by srijs** [comment]
Hi, thanks for reaching out!

I have indeed not much time I can put towards this project right now, so I think it would be great someone else to jump in and push this further.

I'll also add a note to the readme to make this more visible.

**Comment by DCjanus** [comment]
I'm trying to rewrite this project with `async/await`, it's pretty fun.use badge::{Badge, BadgeOptions};

**Comment by berkus** [comment]
@srijs Do you have time to review and approve pull-requests? I might have some time to maintain, but for starters I would like to submit a few patches if you have time to review.

**Comment by berkus** [comment]
Do you have time to review and approve pull-requests? I might have some time to maintain, but for starters I would like to submit a few patches if you have time to review.

**Comment by DCjanus** [comment]
I really like this project, and I noticed that this project has not been updated for a long time. I tried to fix some bug and upgrade dependencies, but because this project is using nightly, I can't build it.

I've rewrited this in stale Rust, there are some tiny feature to be done, would you mind take a look? That's would be really nice. @srijs 

[deps-rs](https://github.com/dcjanus/deps-rs)

For now, It is a separate project, and I hope, if possible, I hope it can be merged into upstream.

**Comment by cecton** [comment]
I have an idea to unblock the situation until the original author comes back.

I'm willing to host it temporarily under my domain, it would be deps.rm.rs instead of deps.rs (so people can update it easily). I can fork the repo, add a few maintainers to be sure nobody waits on me instead, and I have a cluster k8s that can deploy it automatically when master change (so we would be pretty autonomous).

Does any of you want to help me maintain this until the maintainer comes back? After all it's a difficult time, I don't know what happened but I can imagine they have other priorities...

cc @DCjanus @berkus @Feliix42

**Comment by robjtede** [comment]
Not sure if there's been any movement since the last comment here but I'd be interested in being a part of the maintainership going forwards, either on this project or a fork.

**Comment by cecton** [comment]
I just saw the reactions on the message. 

Ok so I will fork and setup CI/CD and add the 4 of you as core maintainers. I won't participate much on the development itself though as I've a few other projects already but I will take care of the dev ops part.

**Comment by cecton** [comment]
I've sent a last email. I have a bad feeling that something bad happened to the author... I hope not but these are strange times... Did anyone try to send an email already?

**Comment by cecton** [comment]
I have found his twitter account https://twitter.com/periping but the dm is locked I think

**Comment by Feliix42** [comment]
By now, I also have a little more time on my hands than in the previous months, so I'm also willing to put some time into this project. I think it'd be nice to pick up the project as a group to minimize the bus factor :)

Should we create an organization for this? I can also try to ping the original author, but he probably has a lot on his plate at the moment.

**Comment by cecton** [comment]
> I think it'd be nice to pick up the project as a group to minimize the bus factor :)

That's definitely my intention! I will setup everything this weekend probably.

> he probably has a lot on his plate at the moment.

I hope so.

**Comment by Feliix42** [comment]
> That's definitely my intention! I will setup everything this weekend probably.

Ah, cool - thanks! If I get to it earlier I can also set this up. A quick glance at the list of Github organizations revealed that `deps-rs` is still free, so we could just use that?

**Comment by cecton** [comment]
Sure! You can create the organization and I will setup the CD with the domain name later

**Comment by Feliix42** [comment]
So I have created [deps-rs](https://github.com/deps-rs) and added @cecton already. I take it that @robjtede also wants to hop on board.
Anybody else interested in maintaining the project with us? (CC @berkus @DCjanus)

@srijs got back to me and said that he would transfer the repository to said organization, so I guess we're (almost) good to go!

**Comment by paolobarbolini** [comment]
I don't know how much time I will have for this, but I would too like to help maintain this project. Count me in

**Comment by jamiebrynes7** [comment]
I'm not too familiar with backend web dev, but I'd been keen to help out where I can 😄

**Comment by berkus** [comment]
@Feliix42 great, count me in!

**Comment by Feliix42** [comment]
I did add you :)

I will close this issue now (for housekeeping reasons), new contributors can still comment down here if they are interested to contribute! 🙂

**Comment by henil** [comment]
@Feliix42 I do like to help too. I am a rust beginner. I could help with the housekeeping/reviewing PR too. Count me in!

**Comment by Feliix42** [comment]
@henil Sure thing! Hop on over to our public matrix channel `#deps.rs:matrix.org` so we can find something for you to work on :)
